src/cookieConsent.js

```diff
@@ -0,0 +1,173 @@
+import React, { useEffect, useState } from "react";
+import { Button, Box, Snackbar, Dialog, DialogTitle, DialogContent, DialogActions, makeStyles } from "@material-ui/core";
+import { DAY_IN_MS } from "./utils/TimeUtils";
+import { clearAnalytics, initAnalytics } from "./utils/analytics";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+import { RouterLink } from "./utils";
+
+export const COOKIE_ACCEPT_LIFETIME_DAYS = 30;
```

Comment:
Maybe this can be increased to 6 months.

[Source](https://www.cookiebot.com/en/gdpr-cookies/#:~:text=Consent%20must%20be%20renewed%20at,frequent%20renewal%2C%20e.g.%206%20months.):
![cookies](https://user-images.githubusercontent.com/50210213/154336773-85122b66-0d62-4358-8e25-9bc64498cc92.png)